docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -138,40 +138,46 @@ The ability to download recordings for offline viewing will be available in a fu
 
 ### Will Teleport be automatically upgraded with each release?
 
-Teleport Cloud follows the Teleport [Upcoming
+Teleport Team and Teleport Enterprise Cloud follow the Teleport [Upcoming
 Releases](../../upcoming-releases.mdx) timelines.
 
-- Major version upgrades for customer Auth and Proxy Servers occur after the
-  first minor release (for example (=teleport.major_version=).1.0) of Teleport.
+- Major version upgrades for cloud Teleport Auth Service and Teleport Proxy Service occur 
```

Comment:
```suggestion
- Major version upgrades for the Teleport Auth Service and Teleport Proxy Service on Teleport Cloud occur 
```
Otherwise we're missing an article